datahub-web-react/src/app/search/autoComplete/AutoCompleteEntity.tsx

```diff
@@ -32,42 +29,82 @@ const ContentWrapper = styled.div`
 `;
 
 const Subtype = styled.span`
-    color: ${ANTD_GRAY[9]};
-    border: 1px solid ${ANTD_GRAY[9]};
+    color: ${ANTD_GRAY_V2[8]};
+    border: 1px solid ${ANTD_GRAY_V2[6]};
     border-radius: 16px;
     padding: 4px 8px;
     line-height: 12px;
     font-size: 12px;
     margin-right: 8px;
 `;
 
+const ItemHeader = styled.div`
+    display: flex;
+    align-items: center;
+    margin-bottom: 3px;
+    gap: 8px;
+`;
+
+const Divider = styled.div`
+    border-right: 1px solid ${ANTD_GRAY_V2[6]};
+    height: 12px;
+`;
+
 interface Props {
     query: string;
     entity: Entity;
+    siblings?: Array<Entity>;
     hasParentTooltip: boolean;
 }
 
-export default function AutoCompleteEntity({ query, entity, hasParentTooltip }: Props) {
+export default function AutoCompleteEntity({ query, entity, siblings, hasParentTooltip }: Props) {
     const entityRegistry = useEntityRegistry();
     const genericEntityProps = entityRegistry.getGenericEntityProperties(entity.type, entity);
-    const platformName = getPlatformName(genericEntityProps);
-    const platformLogoUrl = genericEntityProps?.platform?.properties?.logoUrl;
     const displayName = entityRegistry.getDisplayName(entity.type, entity);
-    const icon =
-        (platformLogoUrl && <PreviewImage preview={false} src={platformLogoUrl} alt={platformName || ''} />) ||
-        entityRegistry.getIcon(entity.type, 12, IconStyleType.ACCENT);
     const { matchedText, unmatchedText } = getAutoCompleteEntityText(displayName, query);
-    const parentContainers = genericEntityProps?.parentContainers?.containers || [];
-    // Need to reverse parentContainers since it returns direct parent first.
-    const orderedParentContainers = [...parentContainers].reverse();
     const subtype = genericEntityProps?.subTypes?.typeNames?.[0];
+    const entities = siblings?.length ? siblings : [entity];
+
+    const platformNames = entities
+        .map((ent) => {
+            const genericPropsForEnt = entityRegistry.getGenericEntityProperties(ent.type, ent);
+            const platform = genericPropsForEnt?.platform;
+            if (platform?.properties?.displayName) return platform.properties.displayName;
+            if (platform?.name) return capitalizeFirstLetterOnly(platform.name) || '';
+            return '';
+        })
+        .filter(Boolean);
```

Comment:
nit: i feel like this could be pulled out into its own function. Also I see elsewhere that we have `siblingPlatforms` on `genericProperties` that you can maybe use here?

Reply:
Updated! I left the iteration in this component but reused an existing helper function we already have.